frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/MyInfoPanel.tsx

```diff
@@ -50,7 +73,7 @@ export const MyInfoFieldPanel = () => {
               {CREATE_MYINFO_PERSONAL_FIELDS_ORDERED.map((fieldType, index) => (
                 <DraggableMyInfoFieldListOption
                   index={index}
-                  isDisabled={isDisabled}
+                  isDisabled={isDisabled || sgidUnSupported(form, fieldType)}
```

Comment:
Thanks, I don't think we can do that as it's dependent on the `fieldType`, which is only known inside the `map`.

Reply:
Ah I see, okay!

Reply:
Actually, I still think this logic should be extracted out for better maintainability (rather than add a new `||` conditional whenever we have something more to condition on. Maybe we can do a function like this:
```ts
const optionIsDisabled = (form, fieldType) => {
  return isDisabled || sgidUnSupported(form, fieldType)
}
```
This way, we'll only need to change this if we have other changes how we want to determine whether a field is disabled in the future.

Reply:
Alright. I've done that.